README.md

```diff
@@ -91,9 +91,27 @@ Development
 
 [![Build Status](https://travis-ci.org/hickford/MechanicalSoup.svg?branch=master)](https://travis-ci.org/hickford/MechanicalSoup)
 
-### Tests
+You can develop against multiple versions of Python using virtualenv:
+
+    python3 -m venv .virtual-py3 && source .virtual-py3/bin/activate
```

Comment:
I have no experience with virtualenv, but this doesn't work for me:

```
$ python3 -m venv .virtual-py
Error: Command '['/home/moy/dev/MechanicalSoup/.virtual-py/bin/python3', '-Im', 'ensurepip', '--upgrade', '--default-pip']' returned non-zero exit status 1
```

OTOH, the command I find [here](http://docs.python-guide.org/en/latest/dev/virtualenvs/) does work:

```
$ virtualenv -p /usr/bin/python3 my_project
...
Installing setuptools, pip, wheel...done.
$ source ./my_project/bin/activate
```

Is there any reason to use `python3 -m venv` instead of `virtualenv`? My setup may well be broken, but as-is I can't confirm that your instructions are correct.